src/error.rs

```diff
@@ -0,0 +1,433 @@
+// Copyright 2014 The Rust Project Developers. See the COPYRIGHT
```

Comment:
This is a vendored version the stable parts of libstd's error.rs.

I know, "Eww", right? I'm not sure of a better solution though.